### PR TITLE
fix: update cargo.toDebianVersion to handle cmake

### PR DESCRIPTION
**Pull request by diogomatsubara**
For cmake versions we have to use the tweak version component to differentiate between release candidate packages.

### CODE REVIEW

File: src/cargo.ts [hunk 1]
```diff
@@ -368,8 +368,24 @@ export function buildDebian(path: string, target: string, version: string) {
...
+    deb_version = `${version.replace("-", "~")}-${revision ?? 1}`;
+  } else {
+    // check cmake version has tweak component
+    if (version.split(".").length == 4) {
+      if (version.endsWith(".0")) {
+        const pos = version.lastIndexOf(".0");
+        deb_version = `${version.substring(0, pos)}~dev-${revision ?? 1}`;
+      } else if (parseInt(version.substring(version.lastIndexOf(".") + 1)) > 0) {
+        const pos = version.lastIndexOf(".");
+        deb_version = `${version.substring(0, pos)}~${version.substring(pos + 1)}-${revision ?? 1}`;
```
Comment (fuzzypixelz) [inline src/cargo.ts:384]:
```suggestion
        deb_version = `${version.substring(0, pos)}~pre.${version.substring(pos + 1)}-${revision ?? 1}`;
```

File: src/cargo.ts [hunk 2]
```diff
@@ -368,8 +368,24 @@ export function buildDebian(path: string, target: string, version: string) {
  * @returns Modified version.
  */
 export function toDebianVersion(version: string, revision?: number): string {
-  // HACK(fuzzypixelz): This is an oversimplification of the Debian Policy
-  return `${version.replace("-", "~")}-${revision ?? 1}`;
+  let deb_version = version;
```
Comment (fuzzypixelz) [inline src/cargo.ts:371]: Please use camelCase everywhere.

File: src/cargo.ts (hunk 1 above)
Comment (diogomatsubara) [reply src/cargo.ts]: Fixed in 
98062bb

File: src/cargo.ts (hunk 2 above)
Comment (diogomatsubara) [reply src/cargo.ts]: Fixed in 98062bb

**Pull request merged**